src/modules/elixir.rs

```diff
@@ -69,22 +72,35 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     Some(module)
 }
 
-fn get_elixir_version(context: &Context) -> Option<(String, String)> {
-    let output = context.exec_cmd("elixir", &["--version"])?.stdout;
-
+fn get_elixir_version(
+    context: &Context,
+    config: &ElixirConfig,
+) -> Option<(Option<String>, String)> {
+    let output = match config.format.contains("$otp_version") {
```

Comment:
The `StringFormatter` has an API for checking if a variable is used. `$otp_version` is not the only valid synatx.

Reply:
Which one? `get_variables`?

Reply:
That should be it.

Reply:
It becomes like this though unless I'm missing something.

```
let versions = Lazy::new(|| match StringFormatter::new(config.format) {
        Ok(fmt) => match fmt.get_variables().contains("otp_version") {
            true => get_elixir_version(context),
            false => get_elixir_version_short(context),
        },
        Err(_) => return None,
    });

let parsed = StringFormatter::new(config.format).and_then(|formatter| {
```

Which seems over complicated for just trying to check if a variable exists in the format string or not.

Reply:
You can skip using the `Lazy<_>` `versions` at all if `!fmt.get_variables().contains("otp_version")`.